deep-security-agent/meta/main.yml

```diff
@@ -0,0 +1,56 @@
+---
+dependencies: []
+galaxy_info:
+  description: Install, Activate and Manage Trend Micro Deep Security Agent
+  min_ansible_version: 2.4
+  company: Trend Micro
+  author: Deep Security
+  license: MIT
+  galaxy_tags:
+    - trend-micro
+    - deep-security
+    - dsm
+    - dsa
+  platforms:
+    - name: Amazon
+      versions:
+      - 2016.09
+    - name: Ubuntu
+      versions:
+      - 12.04.5
+      - 14.04.5
+      - 16.04.0
+    - name: Debian
+      versions:
+      - 7
+      - 8
+    - name: RHEL
+      versions:
+      - 6.9
+      - 7.3
+    - name: Windows
```

Comment:
We may not need to list all the Windows versions here, maybe "ALL" is enough, and we can get any feedback once there are other usage we didn't expected.

Reply:
Changed to "all".